kola/tests/ignition/v2/passwd.go

```diff
@@ -0,0 +1,160 @@
+// Copyright 2017 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ignition
+
+import (
+	"fmt"
+
+	"github.com/coreos/mantle/kola/cluster"
+	"github.com/coreos/mantle/kola/register"
+	"github.com/coreos/mantle/platform"
+)
+
+func init() {
+	register.Register(&register.Test{
+		Name:        "coreos.ignition.v2.groups",
+		Run:         groups,
+		ClusterSize: 1,
+		UserData: `{
+		             "ignition": { "version": "2.0.0" },
+		             "passwd": {
+		               "groups": [
+		                 {
+		                   "name": "group1",
+		                   "gid":  501
+		                 },
+		                 {
+		                   "name": "group2",
+		                   "gid":  502,
+		                   "passwordHash": "3iy3/i4AN2YR6"
+		                 }
+		               ]
+		             }
+		           }`,
+	})
+	register.Register(&register.Test{
+		Name:        "coreos.ignition.v2.users",
+		Run:         users,
+		ClusterSize: 1,
+		UserData: `{
+		             "ignition": { "version": "2.0.0" },
+		             "passwd": {
+		               "users": [
+		                 {
+		                   "name": "core",
+		                   "passwordHash": "$1$xG18DXSV$8DSq5bJfyy60ZBnYJflbK0"
```

Comment:
Considering this test doesn't need a working password how about use an obviously invalid hash so the machine that spins up doesn't have a publicly known credential.